src/mailstoreobserver.cpp

```diff
@@ -38,6 +38,15 @@
 #include <QDBusConnection>
 #include <QDebug>
 
+namespace {
+
+QVariant remoteAction(const QString &name, const QString &displayName, const QString &method, const QVariantList &arguments = QVariantList())
+{
```

Comment:
At this stage it's temporary, I will get around to updating everything to the new design in a subsequent pass.  However, the current design spec won't show the name of the action anywhere either, so it might end up empty anyway :-/